docs/intro/howto.rst

```diff
@@ -123,23 +153,32 @@ output of those commands.
 
 Paths
 =====
-Let's say I want to create a new folder in my home folder,
+
+Paths look different on Unix-like and Windows systems.
+Most prominently, the *path separator*, i.e., the symbol distinguishing directories in path, are back slashes (``\``) instead of front slashes (``/``).
```

Comment:
```suggestion
Most prominently, the *path separators*, i.e., the symbol distinguishing directories in path, are back slashes (``\``) instead of front slashes (``/``).
```